apis/opts/diskquota.go

```diff
@@ -9,17 +9,21 @@ import (
 func ParseDiskQuota(quotas []string) (map[string]string, error) {
 	var quotaMaps = make(map[string]string)
 
+	if quotas == nil {
+		return nil, fmt.Errorf("invalid format for disk quota: %s", quotas)
```

Comment:
We are trying to build PouchContainer a user-friendly, operator-friendly software, so when returning error we should try to make the error message be much clearer. For this case, operators can only get the error with the message `invalid`, but he has no idea why this is invalid.

Thus from my point of view, maybe   `fmt.Errorf("invalid format for disk quota: quotas cannot be empty", quotas)` will be better. In addition, we try to make `quotas` to be formatted is meaningless, because it is already nil.  WDYT? @hellolijj

Reply:
Yeah, that's right!